packages/app/obojobo-express/server/express_validators.js

```diff
@@ -66,7 +66,10 @@ exports.requireVisitId = check('visitId', 'must be a valid UUID').isUUID()
 exports.requireAssessmentId = check('assessmentId', 'must not be empty')
 	.exists({ checkNull: true, checkFalsy: true })
 	.isString()
-
+exports.requireMultipleAttemptIds = [
+	check('attemptIds', 'must be an array of UUIDs').isArray({ min: 1 }),
```

Comment:
these won't error out if attemptIds in an empty array, but we can deal with that later / it's not a big problem here